qcfractal/interface/tests/test_dataset.py

```diff
@@ -235,12 +234,12 @@ def test_database_history():
                ("energy", "p1", "m1", None, "o1"),
                ("energy", "p1", "m1", None, "o2"),
                ("energy", "p1", "m2", "b3", "o1"),
-               ("gradient", "p1", "m2", None, None)] # yapf: disable
+               ("gradient", "p1", "m2", None, None)]  # yapf: disable
 
     for h in history:
         ds._add_history(driver=h[0], program=h[1], method=h[2], basis=h[3], keywords=h[4])
 
     assert ds.list_records().shape[0] == 5
     assert ds.list_records(program="P1").shape[0] == 4
-    assert ds.list_records(basis=None).shape[0] == 3
-    assert ds.list_records(keywords=None).shape[0] == 1
+    assert ds.list_records(basis="None").shape[0] == 3
```

Comment:
I see, a bit awkward and not a "user first" strategy. Still if someone say `ds.list_records(method="ANI1")` that is rather clear, I think its worth it.